mobile/src/main/java/org/openhab/habdroid/ui/OpenHABWidgetAdapter.java

```diff
@@ -618,6 +618,10 @@ public void onStopTrackingTouch(SeekBar seekBar) {
                             spinnerSelectedIndex = spinnerArray.size() - 1;
                         }
                 }
+                if (spinnerSelectedIndex == -1) {
+                    spinnerArray.add("          ");
```

Comment:
Does this really need to be an arbitrary amount of whitespaces? :/

Reply:
Without whitespaces you need to hit that small arrow to open the dropdown.

Reply:
Hmm, at least for the default sitemap (Widget overview -> Scene selection), the area of the selection widget is big enough to open the selection window, so I don't need to tap on the arrow.

Also: Doesn't this adds an empty element to the selection list? This looks a bit weird :/

(I've changed the logic a bit, so that spinnerSelectedIndex is always -1).

Reply:
It is big enough, because it always displays an value. This PR is for the case that the selection item would be set to  - in this case - "sleeping".